src/components/Exercises.js

```diff
@@ -69,18 +70,18 @@ const Exercises = ()=>{
         <button onClick={navigateExerciseForm}>Add New Exercise</button>
         <ul>
           {exercisesData.exercises.map(curr=>{
-            let currWorkout 
-            exercisesData.workouts.forEach((item)=>{
-              if(item._id === curr.workout){
-                currWorkout = item
-              }
-            })
+            // let currWorkout 
+            // exercisesData.workouts.forEach((item)=>{
+            //   if(item._id === curr.workout){
+            //     currWorkout = item
+            //   }
+            // })
         
             return(
               <li >
                 <h3>Title: {curr.title}</h3>
                 <p>Category: {curr.category}</p>
```

Comment:
I am a proponent of being fairly direct with variable names and, as such, would just call this `currentWorkout`. To me, the less added mental overhead, the better. I think generally most people would understand what was meant by `curr`, but the more straightforward, the better.